### PR TITLE
group_set_user_notification_level

### PR DESCRIPTION
**Pull request by kimardenmiller**
- Refactored method
- Created tests & example

### CODE REVIEW

File: lib/discourse_api/api/groups.rb [hunk 1]
```diff
@@ -115,9 +115,8 @@ def group_members(group_name, params = {})
         response.body['members']
       end
 
-      def group_set_user_notification_level(group, user_id, notification_level)
-        response = post("/groups/#{group}/notifications?user_id=#{user_id}&notification_level=#{notification_level}")
-        response
+      def group_set_user_notify_level(group, user_id, notification_level)
```
Comment (oblakeerickson) [inline lib/discourse_api/api/groups.rb]: Could you explain the reasoning behind changing the method name from "notification" to "notify"?

Unless there is a good reason we should probably keep it "notification" to not break anything for existing users?

**Comment by kimardenmiller** [comment]
My editor inspections says it was inspired by this  ["Ruby Style Guide"](https://github.com/rubocop-hq/ruby-style-guide#syntax) to limit method names to 30 long.

File: lib/discourse_api/api/groups.rb (hunk 1 above)
Comment (kimardenmiller) [reply lib/discourse_api/api/groups.rb]: @oblakeerickson what direction would you like to go here so we can close out this PR?

Comment (oblakeerickson) [reply lib/discourse_api/api/groups.rb]: Oh sorry for letting this go so long, let's not change the method name.

**Pull request closed without merging**